packages/zero-runtime/src/processors/styled.ts

```diff
@@ -387,38 +385,46 @@ export class StyledProcessor extends BaseProcessor {
     }
     const value = values.get(this.componentMetaArg.ex.name) as ComponentMeta;
     const { themeArgs: { theme } = {} } = this.options as IOptions;
-    if (!value.name || !theme) {
+    if (!value.name || !value.slot || !theme) {
       return;
     }
     const componentData = (theme as Theme).components?.[value.name];
     if (!componentData) {
       return;
     }
+
     if ('styleOverrides' in componentData) {
-      const overrides = componentData.styleOverrides;
+      const overrides = componentData.styleOverrides as Record<string, unknown> | undefined;
```

Comment:
```suggestion
      const overrides = (componentData.styleOverrides || {}) as Partial<Record<string, Interpolation<unknown>>>;
```

Each slot's value should be typed as `Interpolation` from `@emotion/styled`. It already includes CSSObject, Function and ArrayInterpolation.

Reply:
Ok. So in this particular case, only CSSObject will be supported since to support Interpolation, you'll need to author the css using

```
css`
  ...
`;
```
which won't be available in config.